### PR TITLE
Moved common utilities into the educationplatform static

### PR DESCRIPTION
**Pull request by barnettwilliam**
Moved common utilities into the educationplatform static. 

Updated npm build script to copy files to the dist folder to simplify deployment.

### CODE REVIEW

File: platform/public/common/utility.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    
+                {
+    
+                    "id": "console",
+    
+                    "name": "console",
+    
+                    "panelclass": "ConsolePanel",
+    
+                    "icon": "images/console.gif"
```
Comment (szschaler) [inline platform/public/common/utility.json:29]: Where does this image file live?

**Review by szschaler — APPROVED** [review]
Would be good to cross-reference https://github.com/mdenet/platformtools/pull/18 from this PR for documentation

File: platform/public/common/utility.json (hunk 1 above)
Comment (barnettwilliam) [reply platform/public/common/utility.json:29]: @szschaler  when direct the path used it is relative to the playground base url so it'll be http://127.0.0.1/8080/images/console.gif.

The other way of referencing icons is by using the css id which are loaded relative to the location of the css e.g. the tool static urls.

Comment (szschaler) [reply platform/public/common/utility.json:29]: Thanks for clarifying.

**Pull request merged**